client/web/src/repo/commits/RepositoryCommitsPage.tsx

```diff
@@ -141,20 +142,25 @@ export const RepositoryCommitsPage: React.FunctionComponent<React.PropsWithChild
     return (
         <div className={styles.repositoryCommitsPage} data-testid="commits-page">
             <PageTitle title="Commits" />
-            <FilteredConnection<GitCommitFields, Pick<GitCommitNodeProps, 'className' | 'compact' | 'wrapperElement'>>
-                className={styles.content}
-                listClassName="list-group list-group-flush"
-                noun="commit"
-                pluralNoun="commits"
-                queryConnection={queryCommits}
-                nodeComponent={GitCommitNode}
-                nodeComponentProps={{ className: 'list-group-item', wrapperElement: 'li' }}
-                defaultFirst={20}
-                autoFocus={true}
-                history={props.history}
-                hideSearch={true}
-                location={props.location}
-            />
+            <div className={styles.content}>
+                <H1>View Commits From This Repository</H1>
```

Comment:
```suggestion
                <H1>View commits from this repository</H1>
```

Matches compare page (sentence case)